arch/arm/Kconfig

```diff
@@ -789,6 +789,13 @@ config ARM_THUMB
 	default n
 	depends on ARCH_ARMV7A
 
+config ARM_WFE_SEV
+	bool
+	default y
+	depends on ARCH_ARMV7A && SMP
```

Comment:
it's better to let ARCH_ARMV7A select ARM_WFE_SEV like ARM_HAVE_MPU_UNIFIED

Reply:
Should we add 'select ARM_HAVE_WFE_SEV' to ARCH_CHIP_IMX6 instead of ARCH_ARMV7A?

Reply:
since WFE/SEV isn't optional feature on ARMV7A, I think it's fine to put 'select ARM_HAVE_WFE_SEV' in ARCH_ARMV7A. For old ARM arch and armv6-m, we may need put the selection to the chip level.

Reply:
OK.